### ISSUE TITLE
Localize clue references

### ISSUE THREAD

**Issue by davidmarchand** [mdirolf/crosshare#344]
Currently, clues and comments correctly recognize strings like **17D** or **3A** as references to other clues, with the appropriate underline and hover text. But D for Down and A for Across are English specific. Could these be localized?

In Spanish crosswordry, for example, we say **horizontal** and **vertical** instead of **across** and **down**, so **17D** and **3A** would be **17V** and **3H**.

Ideally, when solving a Spanish-language crossword, a clue with **4H** in its text would link to the corresponding clue. Right now it just shows "4H" as plain text, many examples here: https://crosshare.org/crosswords/luuclOLFRgGiKRI3TlDa

My preference would be for both notation systems to work, regardless of which localization of Crosshare I'm currently on, since I'd like to solve English and Spanish language crosswords comfortably without switching the site's language. I can see how accomodating every language's conventions could be a problem though. Unless every other language that uses the Latin alphabet uses H and V as well, which wouldn't surprise me.

This isn't high priority, but it'd be a nice touch.

**Comment by mdirolf** [comment]
If there is a reasonable `%` of languages for which H + V work then I think we could just make both notations work. Otherwise it's a bit tricky because we want the treatment to match the commenter's language, rather than the viewers (I think). For example - Sommersmith posted a clue a while back where the crosses were in English and the downs in Spanish. That puzzle could reasonably be solved by an english speaking solver at the english version of the site or a spanish speaking solver at the spanish version. Ideally the english speaker could comment in english w/ 4A and the spanish speaker could comment in spanish with 4H and both would get references. But if the english speaker types 4H it's not likely to be a reference.

I'm not sure if that makes any sense or not. One possibility is that most browsers (but not safari!) have` a "detect language" functionality where you can pass a string and get back a guess of the language. Crosshare could treat each comment individually and try to detect its language. If it detects spanish it uses 4H/4V but otherwise sticks with 4A/4D.

**Comment by davidmarchand** [comment]
Yeah it's hard to even imagine a perfect solution. We should probably just ask the other translators about their language's notation just in case H&V truly are as universal as I fancy them to be.
